### PR TITLE
[MASFRES-67] Add IT for excluded ".flattened-pom.xml"

### PR DESCRIPTION
**Pull request by kwin**

### CODE REVIEW

File: source-release/pom.xml [hunk 1]
```diff
@@ -141,6 +141,7 @@ under the License.
                       <value>${project.build.directory}/it-repo</value>
                     </property>
                   </systemProperties>
+                  <argLine>-Dmaven.repo.local="${project.build.directory}/it-repo"</argLine>
```
Comment (michael-o) [inline source-release/pom.xml:144]: IS this relevant to the IT? Maybe it should be separate?!

Comment (kwin) [reply source-release/pom.xml:144]: This was a temporary test for https://issues.apache.org/jira/browse/MASFRES-70.

Comment (slawekjaranowski) [reply source-release/pom.xml:144]: system property set by failsafe should works - you don't need use argLine
By the way - we should first try to upgrade verifier to 2.0.0-M1

Comment (kwin) [reply source-release/pom.xml:144]: It doesn't as outlined in https://issues.apache.org/jira/browse/MASFRES-70 and https://issues.apache.org/jira/browse/MASFRES-70.

**Review by michael-o — APPROVED** [review]
